app/models/pdc_metadata/resource.rb

```diff
@@ -65,6 +66,45 @@ def new_from_json(json_string)
         resource
       end
 
+      def resource_type_general_options
```

Comment:
@jrgriffiniii I wonder if you could utilize `Datacite::Mapping::ResourceTypeGeneral.to_a` instead of hard coding these?